README_ml.md

```diff
@@ -0,0 +1,51 @@
+# Data Scientist Technical Assessment
+
+## Expectations
+
+1. We don’t expect you to spend more than 120 minutes on these exercises, so just work through the questions within the time limit and see what you complete 
+2. In the interest of respecting your free time, we suggest limiting any packages or modules imported to those referenced in the job description
+3. We’re not looking for award-winning visualizations, if you chose to generate a visualization feel free to use the default plot style
+4. We think the provided data is sufficient to answer some really interesting questions and do not expect you to incorporate any additional data sources 
+5. We don’t expect you to be familiar with our filing data - please make assumptions, but be sure to document them in your code so that we can follow your thought process.
+6. When you’re ready, send us back a markdown file with your work (either RMarkdown or Jupyter notebook) or an R or Python file with your code  + comments to help us follow your work
+
+
+### Exercise One: SQL Refactor
+**Objective**: Refactor [this SQL Query](https://docs.google.com/document/d/1idPKfrhwhuw5C5qDMwWun0MI5wQaBI288RFMafcTDos/edit?usp=sharing) so that it is easily readable by your teammates and fix any syntax errors you come across. 
+
+**Background**: 
+  - We store our data in a redshift database, and while we’ve done a lot of good work modeling this raw resource to help ourselves and teams across the organization quickly answer basic questions about fundraising patterns on our site through a BI tool, many of the deeper explorations our team does for the organization starts with manipulating the data stored in our redshift cluster using SQL. 
+  - This query is a messy version of the query we used to pull the data used in exercise two of this take home. All of the where clauses, and table sources are correct and don’t need to be updated, but this is obviously not a great work product to share with colleagues - it’s hard to scan for quick understanding of the work, has inconsistent syntax, and there are some syntactical errors that would cause this query run to fail. 
+
+**Suggested Time**: 30 minutes 
+
+**Notes**:
+  - If you are not familiar with Common Table Expressions (CTEs) check out [this resource](https://learnsql.com/blog/what-is-common-table-expression/)
+  - Everyone has different SQL styles and we are intentionally not sharing our internal style guide, so prioritize consistency over a specific style 
+  - Highly suggested that you leave comments throughout the file to explain your work, express uncertainty, or ask questions
+
+
+### Exercise Two: Model Building and Deployment
+**Objective**: Build a predictive model from ActBlue’s (publicly available) FEC filing
```

Comment:
Do you want to add a line or two about your expectations around the predictive model they build since they only have 90 minutes and also considering the data set they are working with.